plugin/core/message_request_handler.py

```diff
@@ -0,0 +1,93 @@
+import mdpopups
+from .rpc import Client
+from .typing import Any, List, Callable
+from .protocol import Response
+from sublime import View
+
+
+class MessageRequestHandler(object):
```

Comment:
You can leave out the fact that `MessageRequestHandler` inherits from `object` -- that is old-style python2 syntax https://stackoverflow.com/questions/4015417/python-class-inherits-object